### DISCUSSION TITLE
Icon shape

### DISCUSSION THREAD

**Discussion by abhishekdhadwal** [LawnchairLauncher/lawnchair#3500] [Ideas]
Please add pebble style icon shape
![Screenshot_20230605_061637_Smart Launcher](https://github.com/LawnchairLauncher/lawnchair/assets/41069780/c70f4e9a-7f3e-4562-a32a-5f59789e61e4)

**Comment by Kuijano-Evindass** [comment]
I'd also like hexagons, and other polygons.

**Comment by validcube** [comment]
Hello, 

Pebble icon shape have been added to Lawnchair 16, you can try them now when using the latest version of Lawnchair Nightly.
